modules/atomic-widgets/parsers/props-parser.php

```diff
@@ -46,12 +48,15 @@ public function validate( array $props ): array {
 
 			if ( ! $is_valid ) {
 				$this->errors_bag[] = $key;
+				$validated[ $key ] = false;
```

Comment:
The `validated` is with only the flags of the validity of each field, while `sanitized` holds the actual values that should be used/stored, so they serve different purposes
I could maybe unify them into one object of `value`, `sanitized_value`, `is_valid`, did you mean something like that? It could even have error messages, instead of the `error_bag` we use